components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -757,6 +766,695 @@ impl Interpolate for LengthOrNone {
         TransformList(Some(result))
     }
 
+    /// https://drafts.csswg.org/css-transforms/#Rotate3dDefined
+    fn rotate_to_matrix(x: f32, y: f32, z: f32, a: SpecifiedAngle) -> ComputedMatrix {
+        let rad = a.radians();
```

Comment:
You're right. I did this because of `sq` is using it as full. But since last changes, we don't need this and I changed it as you suggested.